tests/mgbench/workloads/disk_pokec.py

```diff
@@ -0,0 +1,449 @@
+    #     )
+
+    # OK
+    def benchmark__arango__allshortest_paths(self):
+        vertex_from, vertex_to = self._get_random_from_to()
+        return (
+            "MATCH (n:User {id: $from}), (m:User {id: $to}) WITH n, m "
+            "MATCH p=(n)-[*allshortest 2 (r, n | 1) total_weight]->(m) "
+            "RETURN extract(n in nodes(p) | n.id) AS path",
+            {"from": vertex_from, "to": vertex_to},
+        )
+
+    # Our benchmark queries
+
+    # OK
+    def benchmark__create__edge(self):
+        vertex_from, vertex_to = self._get_random_from_to()
+        return (
+            "MATCH (a:User {id: $from}), (b:User {id: $to}) " "CREATE (a)-[:TempEdge]->(b)",
+            {"from": vertex_from, "to": vertex_to},
+        )
+
+    # OK
+    def benchmark__create__pattern(self):
+        return ("CREATE ()-[:TempEdge]->()", {})
+
+    # OK
+    def benchmark__create__vertex(self):
+        return ("CREATE ()", {})
+
+    # OK
+    def benchmark__create__vertex_big(self):
+        return (
+            "CREATE (:L1:L2:L3:L4:L5:L6:L7 {p1: true, p2: 42, "
+            'p3: "Here is some text that is not extremely short", '
+            'p4:"Short text", p5: 234.434, p6: 11.11, p7: false})',
+            {},
+        )
+
+    # OK
+    def benchmark__aggregation__count(self):
+        return ("MATCH (n) RETURN count(n), count(n.age)", {})
+
+    # OK
+    def benchmark__aggregation__min_max_avg(self):
+        return ("MATCH (n) RETURN min(n.age), max(n.age), avg(n.age)", {})
+
+    # NOT OK
+    # def benchmark__match__pattern_cycle(self):
+    #     return (
+    #         "MATCH (n:User {id: $id})-[e1]->(m)-[e2]->(n) " "RETURN e1, m, e2",
+    #         {"id": self._get_random_vertex()},
+    #     )
+
+    # NOT OK
+    # def benchmark__match__pattern_long(self):
+    #     return (
+    #         "MATCH (n1:User {id: $id})-[e1]->(n2)-[e2]->" "(n3)-[e3]->(n4)<-[e4]-(n5) " "RETURN n5 LIMIT 1",
+    #         {"id": self._get_random_vertex()},
+    #     )
+
+    # OK
+    def benchmark__match__pattern_short(self):
+        return (
+            "MATCH (n:User {id: $id})-[e]->(m) " "RETURN m LIMIT 1",
+            {"id": self._get_random_vertex()},
+        )
+
+    # OK
+    def benchmark__match__vertex_on_label_property(self):
+        return (
+            "MATCH (n:User) WITH n WHERE n.id = $id RETURN n",
+            {"id": self._get_random_vertex()},
+        )
+
+    # OK
+    def benchmark__match__vertex_on_label_property_index(self):
+        return ("MATCH (n:User {id: $id}) RETURN n", {"id": self._get_random_vertex()})
+
+    # OK
+    def benchmark__match__vertex_on_property(self):
+        return ("MATCH (n:User {id: $id}) RETURN n", {"id": self._get_random_vertex()})
+
+    # OK
+    def benchmark__update__vertex_on_property(self):
+        return (
+            "MATCH (n {id: $id}) SET n.property = -1",
+            {"id": self._get_random_vertex()},
+        )
+
+    # Basic benchmark queries
```

Comment:
You can delete all the basic queries (`benchmark__basic`), they were c/p for the benchgraph, have the same group and have _type in the name. There were two reasons:

- not to change the json structure 
- and to have a query type for benchgraph. 

The first batch of queries contained everything that is on Grafana.

Reply:
basic queries are c/p of what? Can you explain a bit more what you meant, not sure I understood?

Reply:
Basic queries are c/p from the pokec.py (arango, create, aggregate). They were named `basic`, to drop Arango group and added the `_type` extension, we needed this for benchgraph. We did that to avoid changing the existing queries JSON paths (as I mentioned) . If you take a look, you will notice that basic queries are identical to (arango + create + aggregate). Only the shortest path is missing due to not being able to run on Neo4j. 

So feel free to delete them.